who_is_active.sql

```diff
@@ -1988,8 +2081,18 @@ BEGIN;
 									END COLLATE SQL_Latin1_General_CP1_CI_AS
 								) AS program_name,
 								MAX(sp2.dbid) AS database_id,
-								MAX(sp2.memusage) AS memory_usage,
-								MAX(sp2.open_tran) AS open_tran_count,
+								' + 
+								CASE
+									WHEN (@get_memory_grant_info = 1 AND @sql_version >= 1001600) THEN
+										'MAX(mg.used_memory_kb) AS memory_usage,

```

Comment:
What if we simply COALESCE this with the `sp2.memusage`, so it's never nullable?